hoot-core-test/src/test/cpp/hoot/core/test/ConflateCaseTestSuite.cpp

```diff
@@ -92,7 +93,8 @@ void ConflateCaseTestSuite::loadDir(QString dir, QStringList confs)
 
     if (ignore)
     {
-      LOG_WARN("Disabling: " + path);
+      if (!_hideDisableTests)
+        LOG_WARN("Disabling: " + path);
```

Comment:
Disable the warning if required.  This keeps the parallel jobs from spitting this warning out multiple times.